### DISCUSSION TITLE
saf-training/courses/advanced/09

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#282] [Ideas]
# saf-training/courses/advanced/09

GitHub Actions Let's create a GitHub Action workflow to define our pipeline. The Workflow File Pipeline orchestration tools are usually configured in a predefined workflow file,...

https://mitre.github.io/saf-training/courses/advanced/09.html

**Comment by karikarshivani** [comment]
Improve readability with multi-line commands in pipeline tasks

**Comment by Erndogg67** [comment]
Just a head's up that you have a small typo within the "Next Steps" section:

"You may also see a yellow circle to indicate that the pipeline has not completed yet, or a red X mark to indicate an errr, depending on the status of your pipeline when you examine it"

**Reply by Erndogg67** [reply]
My apologies - it is "The Next Step" section
